### PR TITLE
fix: project creation

### PR DESCRIPTION
**Pull request by Jisay**
# Project Creation

### :bug: Bug Fix

596f5a0 - fix(pci.projects.project.creating): redirection after creation complete
2acb4c4 - fix(pci.projects.creation): redirect to creating page when credit OK
10c8b0f - fix(pci.projects.new): one shot payment when fraud detection

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/payment/payment.routing.js [hunk 1]
```diff
@@ -5,6 +5,22 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider
     .state('pci.projects.new.payment', {
       url: '/payment?mode&credit&voucher&hiPayStatus&paypalAgreementStatus',
+      redirectTo: (transition) => {
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/new/payment/payment.routing.js]: You can use destructuring assignment for instance:

```js
redirectTo: (transition) => {
  const { hiPayStatus, mode, projectId } = transition.params();

   if (hiPayStatus === 'success' && mode === 'credits' && projectId) {
    return {
      state: 'pci.projects.project',
      params: {
        projectId,
      },
    };
  }

   return null;
},
```

File: packages/manager/modules/pci/src/projects/new/new.routing.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ export default /* @ngInject */ ($stateProvider) => {
         getStepByName: /* @ngInject */ steps => stepName => find(steps, {
           name: stepName,
         }),
-        getStateLink: /* ngInject */ ($state, getCurrentStep) => (action) => {
+        getStateLink: /* ngInject */ ($state, getCurrentStep) => (action, inherit = true) => {
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/new/new.routing.js]:
```suggestion
        getStateLink: /* @ngInject */ ($state, getCurrentStep) => (action, inherit = true) => {
```

File: packages/manager/apps/public-cloud/src/index.js [hunk 3]
```diff
@@ -73,6 +73,7 @@ angular
   })
   .run(/* @ngInject */ ($state) => {
     $state.defaultErrorHandler((error) => {
+      console.log(error);
```
Comment (FredericEspiau) [inline packages/manager/apps/public-cloud/src/index.js]: 🎉

File: packages/manager/modules/pci/src/projects/new/new.routing.js [hunk 4]
```diff
@@ -97,8 +99,30 @@ export default /* @ngInject */ ($stateProvider) => {
           PCI_REDIRECT_URLS,
           `${coreConfig.getRegion()}.paymentMethods`,
         ),
-        newProjectInfo: /* @ngInject */ ($timeout, coreConfig, ovhPaymentMethod,
+        project: /* @ngInject */ ($transition$, PciProjectNewService) => {
+          if ($transition$.params().projectId) {
+            return PciProjectNewService
+              .getProject($transition$.params().projectId);
+          }
+          return null;
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.routing.js]:
```suggestion
          return $q.when();
```

if `PciProjectNewService.getProject` returns a `Promise`

**Pull request merged**